mmv1/third_party/terraform/services/compute/resource_compute_region_instance_group_manager.go.erb

```diff
@@ -848,6 +899,26 @@ func resourceComputeRegionInstanceGroupManagerUpdate(d *schema.ResourceData, met
 		change = true
 	}
 
+  var targetSizePatchUpdate bool
```

Comment:
# github.com/hashicorp/terraform-provider-google/google/services/compute
Error: google/services/compute/resource_compute_region_instance_group_manager.go:818:6: targetSizePatchUpdate declared and not used

Reply:
Moved to the block <% unless version == "ga" -%>
Hopefully it will resolve this.